.gitlab-ci.yml

```diff
@@ -0,0 +1,60 @@
+stages:
+  - test
+  - deploy
+
+test:
+  stage: test
+  image: python:3.8.7-slim-buster
+  before_script:
+    - python -m pip install --upgrade pip
+    - pip install pytest certifi
+    - if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+    - python setup.py install
+    # reduce the default SSL security level to get around a misconfiguration in the Ensembl server
+    # https://github.com/Ensembl/ensembl-rest/issues/427
+    # See https://askubuntu.com/a/1233456
+    - sed -i 's/DEFAULT@SECLEVEL=2/DEFAULT@SECLEVEL=1/' /usr/lib/ssl/openssl.cnf
+  script:
+    - PYTHONPATH=. pytest tests
+  environment:
+    name: test-env
+  only:
+    - external_pull_requests
```

Comment:
Should the pipeline be triggered also for external PRs?

Reply:
This mean pull request in Github so yes I think it should.
It doesn't actually work for pull request from fork so it has limited use anyway.